src/modules/formBuilder/components/itemEditor/FormEditorItemProperties.tsx

```diff
@@ -77,15 +79,14 @@ const FormEditorItemProperties: React.FC<FormEditorItemPropertiesProps> = ({
     formState: { isDirty, dirtyFields },
   } = handlers;
 
-  // Item type is not allowed to change in this form
-  const [itemTypeValue] = useState<ItemType>(initialItem.type);
+  const itemTypeValue = watch('type');
```

Comment:
[just for my own understanding] What's the reason for `itemTypeValue` to now be a watched field? It's still the case that we don't yet allow editing the item type, right?